src/commonMain/kotlin/com/charleskorn/kaml/YamlInput.kt

```diff
@@ -32,49 +32,50 @@ import kotlinx.serialization.modules.SerializersModule
 @OptIn(ExperimentalSerializationApi::class)
 public sealed class YamlInput(
     public val node: YamlNode,
+    public val yaml: Yaml,
```

Comment:
Why is this `yaml: Yaml` property required? Is this so that custom deserializers can access it?

Reply:
Yes, that's correct. I explained above that the Json implementations in kotlinx.serialization expose the `Json` instances in the same way.